src/components/InventoryManager/ListingSteps/PricesStep.js

```diff
@@ -0,0 +1,248 @@
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ListingInputField from '../ListingInputField';
+
+const useStyles = makeStyles({
+  heading: {
+    fontFamily: 'Work Sans',
+    fontWeight: '700',
+    textDecoration: '#53AA48 solid underline 2px',
+    textUnderlinePosition: 'under',
+    fontSize: '1.4rem',
+  },
+  subheading: {
+    fontFamily: 'Work Sans',
+    fontWeight: '600',
+    fontSize: '1rem',
+  },
+  title: {
+    fontFamily: 'Work Sans',
+    fontSize: '2.3rem',
+    fontWeight: '400',
+  },
+  text: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    fontWeight: '400',
+  },
+  labels: {
+    fontFamily: 'Work Sans',
+    fontWeight: '600',
+    fontSize: '1.05rem',
+  },
+  unitLabel: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    fontWeight: '400',
+    paddingLeft: '.5rem',
+  },
+});
+
+export default function PricesStep({
+  setListingRecord, listingRecord,
+}) {
+  const classes = useStyles();
+  const questions = [
+    '1. What is the standard price of your produce?',
+    '2. What is the agency price of your produce?',
+  ];
+  const onChangePrice = (e, newValue, target) => {
+    setListingRecord({ ...listingRecord, [target]: newValue });
+  };
+  const validPrices = `${listingRecord['has master units'] ? 'Master Unit/' : ''}Pallet${listingRecord['has master pallets'] ? '/Master Pallet' : ''}`;
+  const getPrices = (basePrice) => {
+    const prices = {};
+    if (listingRecord['has master units']) {
+      prices.masterUnitPrice = listingRecord['grouped produce type per master unit'] * basePrice;
+    } else {
+      prices.masterUnitPrice = 0;
+    }
+    prices.palletPrice = prices.masterUnitPrice * listingRecord['master units per pallet'] + basePrice * listingRecord['grouped produce type per pallet'];
+    prices.masterPalletPrice = prices.palletPrice * listingRecord['pallets per master pallet'];
+    return prices;
+  };
+
+  const standardPrices = getPrices(listingRecord['standard price per grouped produce type']);
+  const agencyPrices = getPrices(listingRecord['agency price per grouped produce type']);
+  return (
+    <>
+      <Grid spacing={3} container>
+        <Grid item container xs={12}>
+          <Typography gutterBottom variant="h1" component="h1" className={classes.title}>Step 3: Prices</Typography>
+        </Grid>
+        <Grid item container xs={12}>
+          <Typography variant="h2" component="h2" className={classes.heading}>{questions[0]}</Typography>
+        </Grid>
+        <Grid container item xs={12}>
+          <Grid container spacing={1} item xs={12}>
+            <Grid item xs={12}>
+              <Typography className={classes.subheading} variant="h2" component="h2">
+                Standard Price per Grouped Produce Type
+              </Typography>
+            </Grid>
+            <Grid container item xs={12} alignItems="center">
+              <ListingInputField
```

Comment:
Make sure that this listing input field cannot be negative or 0 when you move forward